### PR TITLE
NoSSR sunshine sidebar to make site faster for admins

### PR DESCRIPTION
**Pull request by Will-Howard**
I think it's bad for the site to be a lot slower for admins because it throws off our sense of whether the site has acceptable performance, also it's just annoying. This makes the front page load a lot faster against the prod db locally, there may be other admin-only things that I'm not aware of though some I'm not sure if this will result in total parity between admin and non-admin load times

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203099306884608) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/Layout.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ import type { ToCData, ToCSection } from '../server/tableOfContents';
 import { ForumOptions, forumSelect } from '../lib/forumTypeUtils';
 import { userCanDo } from '../lib/vulcan-users/permissions';
 import { getUserEmail } from "../lib/collections/users/helpers";
+import NoSSR from 'react-no-ssr';
```
Comment (jpaddison3) [inline packages/lesswrong/components/Layout.tsx:26]: Our codebase seems divided between material-ui-core and react-no-ssr for importing NoSSSR

**Comment by jpaddison3** [comment]
I think the test failure is real.

**Review by jpaddison3 — COMMENTED** [review]
Fix test

**Comment by Will-Howard** [comment]
Update: I gave the cypress tests another go now they have been fixed up a lot by @oetherington and this seems to be working now 🤷‍♂️ . Seeing as I could never replicate any real error manually I'd be happy to merge this without trying to work out exactly what was going wrong

**Review by jpaddison3 — APPROVED** [review]

**Pull request merged**